dpctl/tensor/_sorting.py

```diff
@@ -74,10 +92,33 @@ def sort(x, /, *, axis=-1, descending=False, stable=True):
             axis,
         ]
         arr = dpt.permute_dims(x, perm)
+    if kind is None:
+        kind = "stable"
+    if not isinstance(kind, str) or kind not in [
+        "stable",
+        "radixsort",
+        "mergesort",
+    ]:
+        raise ValueError(
+            "Unsupported kind value. Expected 'stable', 'mergesort', "
+            f"or 'radixsort', but got '{kind}'"
+        )
```

Comment:
Logic needs to be changed so that if `stable=True`, `"stable"` is always selected regardless of input `kind`.

This is because array API specifies the `stable` keyword.

Same for `argsort`

Reply:
Presently, we so not implement unstable sorting algorithms. They are all stable. `kind="stable"` is a heuristic algorithm which switches between radix and merge parallel sorting algorithms based on the data type.

Reply:
I see, it's a bit confusing given the `stable` keyword but it seems this is done by NumPy as well, so I don't think any change is necessary.